magento/magento2
# Fix disappearing navigation arrows in fotorama zoom

### Description (*)
Fix for dissapearing navigation arrows in Fotorama zoom

### Fixed Issues (if relevant)
magento/magento2#18585: Navigation arrows zoomed fotorama disappear

### Manual testing scenarios (*)
1. Go to category Women -> Tops -> Jackets
2. Open product "Olivia 1/4 Zip Light Jacket"
3. Click on the large product image on the left
4. Check that navigation arrows are visible
5. Click on a thumbnail
6. Check that navigation arrows keep visible

### Contribution checklist (*)
